zipkin-web/src/main/scala/com/twitter/zipkin/web/Main.scala

```diff
@@ -38,10 +39,11 @@ trait ZipkinWebFactory { self: App =>
   LoggerFactory.getLogger(org.slf4j.Logger.ROOT_LOGGER_NAME)
     .asInstanceOf[Logger].setLevel(Level.toLevel(logLevel))
 
- /**
-  * Initialize an HTTP proxy service, targeting the query host.
-  */
-  def newWebServer(): Service[Request, Response] = Http.newService(queryDest())
+  /**
+    * Initialize an HTTP proxy service, targeting the query host.
+    */
+  def proxyClient(): Service[Request, Response] = Http.client.configured(param.Label("zipkin-web"))
```

Comment:
basically it was a small regression, as the annotations should be logged under the local service name (in this case zipkin-web)